src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import './ChatEntry.css'
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const Chatlog = ({entries, toggleLike}) => {
+    const  getEntries = () => {
+        return entries.map((entry, index) => {
+        return (
+            <ChatEntry 
+            key={index}
+            id={entry.id}
+            sender={entry.sender} 
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            liked={entry.liked}
+            toggleLike={toggleLike}
+            />
+        );
+    });
+}
+    return <main>
+        <section className='chat-log'>
+            {getEntries()}
+        </section>
+        </main>
+};
+
+Chatlog.propTypes = {
+    entries: PropTypes.arrayOf (
+        PropTypes.shape (
```

Comment:
Nice use of PropTypes.shape for more detail on what data is expected for this component =]